server/src/models/GameState.ts

```diff
@@ -97,6 +105,26 @@ export class GameState {
                 opponent.diedAt = now
             }
         }
+
+        for (const houseId of Object.keys(this.houses)) {
+            const opponentHouse = this.houses[houseId]
+            if (opponentHouse.hp <= 0) {
+                continue
+            }
+
+            if (opponentHouse.team == hero.team) {
+                continue
+            }
+
+            const dx = opponentHouse.position.x - hero.position.x
+            const dy = opponentHouse.position.y - hero.position.y
+            const distanceSquared = dx * dx + dy * dy
+            if (doubleHeroRadiusSquared < distanceSquared) {
```

Comment:
This needs to use the square of the sum of the hero radius (60) and the house radius (120) because houses are big and you can hit them far away from their center